app/src/main/res/values/strings.xml

```diff
@@ -264,10 +286,21 @@
     <!-- Menu text for setting themes -->
     <string name="label_menu_themes">Themes</string>
    
+   
+    <!-- Theme settings -->
+    <!-- Titles in theme settings panel -->
+    <string name="theme_settings_title">Choose your theme</string>
+    <string name="theme_settings_subtitle">Save some battery by selecting a dark theme.</string>
+   
+    <!-- Dialog for asking if the user want to set as default browser-->
+    <string name="set_as_default_dialog_title">Well done!</string>
+    <string name="set_as_default_dialog_subtitle">Your Firefox Lite is looking great! Do you want to set it as your default browser?</string>
```

Comment:
`Firefox Lite` should be replaced by placeholder (`%1$s`)

Reply:
OK!